src/main/java/hill/weatherconfidence/ingest/model/GeocodeItem.java

```diff
@@ -0,0 +1,74 @@
+package hill.weatherconfidence.ingest.model;
+
+
+/*
+ * [{"name":"Hemel Hempstead","local_names":{"en":"Hemel Hempstead","ru":"Хемел-Хемпстед"},"lat":51.7511784,"lon":-0.472528,"country":"GB","state":"England"}]
+ * */
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import java.util.Objects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class GeocodeItem {
```

Comment:
A model class that the JSON response from the webserver will be deserialised into.